release/docker/config/opensearch/opensearch-docker-entrypoint.sh

```diff
@@ -71,6 +74,25 @@ done < <(env)
 # will run in.
 export OPENSEARCH_JAVA_OPTS="-Dopensearch.cgroups.hierarchy.override=/ $OPENSEARCH_JAVA_OPTS"
 
+##Security Plugin
+SECURITY_PLUGIN="opensearch-security"
+if [ -d "$OPENSEARCH_HOME/plugins/$SECURITY_PLUGIN" ]; then
+    if [ "$DISABLE_INSTALL_DEMO_CONFIG" = "true" ]; then
+        echo "Disabling Install Demo Config for OpenSearch Security Plugin"
```

Comment:
Can we be more clear with this log message? You are checking if security plugin is present or not right? 
Can we say "security plugin is not installed" You are not actually disabling anything here right? Just echo

Reply:
This is not about security plugin but I will improve the msg.